ACE/tests/Message_Queue_Test_Ex.cpp

```diff
@@ -709,6 +709,98 @@ int queue_priority_test (ACE_Message_Queue_Ex<User_Class, ACE_SYNCH>& q)
   return status;
 }
 
+class Queue_Ex_Iterator_No_Lock
+ : public ACE_Message_Queue_Iterator<ACE_SYNCH, ACE_System_Time_Policy>
+{
+  typedef ACE_Message_Queue_Iterator<ACE_SYNCH, ACE_System_Time_Policy> inherited;
```

Comment:
Please make use of `using` instead of `typedef`

Reply:
ok